svm/src/account_loader.rs

```diff
@@ -528,6 +528,10 @@ mod tests {
         fn get_feature_set(&self) -> Arc<FeatureSet> {
             self.feature_set.clone()
         }
+
+        fn add_builtin_account(&self, _name: &str, _program_id: &Pubkey, _must_replace: bool) {
+            todo!()
```

Comment:
There is still a `todo!()` left over here.

Reply:
@LucasSte if this is an optional trait method, maybe have a default implementation of it in the trait definition?

Reply:
The default implementation can be empty. So you don't need to implement for test with a todo in it. If the users of SVM do not need builtin they can ignore it.